conda/core/path_actions.py

```diff
@@ -1129,15 +1133,12 @@ def cleanup(self):
 class RemoveMenuAction(RemoveFromPrefixPathAction):
     @classmethod
     def create_actions(cls, transaction_context, linked_package_data, target_prefix):
-        if on_win:
-            MENU_RE = re.compile(r"^menu/.*\.json$", re.IGNORECASE)
-            return tuple(
-                cls(transaction_context, linked_package_data, target_prefix, trgt)
-                for trgt in linked_package_data.files
-                if bool(MENU_RE.match(trgt))
-            )
-        else:
-            return ()
+        MENU_RE = re.compile(r"^menu/.*\.json$", re.IGNORECASE)
```

Comment:
Would it make sense for this to be moved to the module level?

Reply:
Yea, I think lowercasing it is enough. I just moved the line from one place to the other to minimize the PR diff.

Reply:
Totally understand that.